src/components/NewBook.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+const NewBook = () => {
+  const [newBook, setNewBook] = useState({
+    title: '',
+    author: '',
+  });
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setNewBook({
+      ...newBook,
+      [name]: value,
+    });
+  };
+
+  return (
+    <div className="add-book">
+      <h1>ADD NEW BOOK</h1>
+      <form>
+        <input type="text" name="title" placeholder="Book title" value={newBook.title} onChange={handleInputChange} />
+        <input type="text" name="author" placeholder="Book author" value={newBook.author} onChange={handleInputChange} />
+        <button type="submit">ADD BOOK</button>
+      </form>
+    </div>
+  );
+};
```

Comment:
- [ ]  You did an amazing job to implement the requirement, but when I check in the console, react is complaining about the in put field that, it should have a default value or onchange method set. Fix the issue so that there will be no error in the console.  check here for [more](https://stackoverflow.com/questions/43556212/failed-form-proptype-you-provided-a-value-prop-to-a-form-field-without-an-on).
![image](https://github.com/yashodhicy/BookStore/assets/43877689/ed6b75ac-c747-4231-a6ae-4e14c068bcd9)

Reply:
thank you